src/foundry/client/config.d.ts

```diff
@@ -94,6 +94,21 @@ declare global {
       /** @defaultValue `"fas fa-id-badge"` */
       sidebarIcon: string;
 
+      /**
+       * @defaultValue
+       * ```javascript
```

Comment:
use `typescript` instead, and make it a complete object, I.e. put `{}` around the properties